mappings/net/minecraft/block/Block.mapping

```diff
@@ -103,8 +103,6 @@ CLASS net/minecraft/unmapped/C_mmxmpdoq net/minecraft/block/Block
 		ARG 2 stack
 	METHOD m_fcwpdfta appendStacks (Lnet/minecraft/unmapped/C_lnokcayd;Lnet/minecraft/unmapped/C_rnrfftze;)V
 		COMMENT Appends the stacks of this block shown in the item group to the list.
-		COMMENT
-		COMMENT @see net.minecraft.item.BlockItem#appendStacks(ItemGroup, DefaultedList)
```

Comment:
Is there no alternative?

Reply:
i think we should wait until the new item group stuff is mapped to know for sure

Reply:
I didn't really check, but because of the item-group changes an alternative may not exist or may be unmapped, so as Oro said, we should wait until the new stuff is mapped